packages/data-point/lib/entity-types/entity-hash/resolve.test.js

```diff
@@ -135,7 +135,7 @@ describe('entity.hash.addValues', () => {
       })
     })
   })
-  test('do nothing if mapKeys is empty', () => {
+  test('do nothing if mapKeys is empty | f collection', () => {
```

Comment:
Can we add `It should` to the beginning of this description? I don't think we need the stuff from the `|` on.

Also, I noticed that this description is the same for the others above it, but they are in different `describe` blocks, so I think that's OK.